README.md

```diff
@@ -315,6 +315,10 @@ For complex/professional level work, you have optional full control of the train
 
 ```python
 class LitAutoEncoder(pl.LightningModule):
+    def __init__(self):
+        super().__init__()
+        self.automatic_optimization = False
+
     def training_step(self, batch, batch_idx, optimizer_idx):
         # access your optimizers with use_pl_optimizer=False. Default is True
```

Comment:
this comment has to be removed, right @tchaton?

Reply:
I think the only difference here is if it's true then lightning will handle zero_grad + some hooks + optimizer_step in different backends else it's just a normal pytorch optimizer. @tchaton can you confirm?

Reply:
correct, the comment should be removed

Reply:
done.